src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -18,106 +18,119 @@
+                                                        parent-id={parentId}
+                                                        parent-s-object-type={rdSettings.parentSObjectType}
+                                                        data-id="donorComponent"
+                                                        onerrorevent={handleChildComponentError}
+                                                        if:true={isSettingReady}>
+                        </c-rd2-entry-form-donor-section>
 
-                    <template if:true={isEdit}>
-                        <div class="slds-section slds-is-open">
-                            <h3 class="slds-section__title slds-theme_shade">
-                                <span class="slds-truncate slds-p-horizontal_small">Status Information</span>
-                            </h3>
-                            <div aria-hidden="false" class="slds-section__content">
-
-                                <lightning-layout horizontal-align="spread" multiple-rows="true">
-                                    <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                        <lightning-input-field field-name={fields.status.apiName} required>
-                                        </lightning-input-field>
-                                    </lightning-layout-item>
-
-                                    <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                        <lightning-input-field field-name={fields.statusreason.apiName}>
-                                        </lightning-input-field>
-                                    </lightning-layout-item>
-                                </lightning-layout>
-
-                            </div>
-                        </div>
-                    </template>
+                    </div>
+                </div>
 
+                <template if:true={isEdit}>
                     <div class="slds-section slds-is-open">
                         <h3 class="slds-section__title slds-theme_shade">
-                            <span class="slds-truncate slds-p-horizontal_small">{customLabels.scheduleSectionHeader}</span>
+                            <span class="slds-truncate slds-p-horizontal_small">{customLabels.statusSectionHeader}</span>
                         </h3>
                         <div aria-hidden="false" class="slds-section__content">
 
-                            <lightning-layout horizontal-align="spread" multiple-rows="true">
-                                <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                    <lightning-input-field field-name={fields.amount.apiName} required>
+                            <lightning-layout horizontal-align="spread" multiple-rows="true" if:true={isFormReady}>
```

Comment:
@force2b I forced the error on save by using a negative value in the 'Every' field. I notice the status and status reason is also clear out It seem like the error is caused by lightning-input-field being re-rendered on save. If you remove the if:true={isFormReady} on here and line 76, the field will not be re-rendered

Reply:
Thanks and great suggestion. Worked perfectly.